app/views/provider/sessions/new.html.slim

```diff
@@ -3,7 +3,7 @@
 - flash_messages = (flash || []).map { |f| {type: f[0], message: f[1]}}
 - is_master_account = domain_account.master?
 div#pf-login-page-container data-login-props={redirectUrl: (session[:return_to].nil? ? "null" : (request.protocol + request.host_with_port + session[:return_to])),
-  enforceSSO: false,
+  enforceSSO: @provider.settings.enforce_sso?,
   authenticationProviders: authentication_providers,
   flashMessages: flash_messages,
   show3scaleLoginForm: @presenter.show_username_password_related_content?,
```

Comment:
seems like this is already taking care of it: the presenter method references `@domain_account.settings.enforce_sso?`

Reply:
Done!